docs/network-flow-visibility.md

```diff
@@ -237,8 +241,13 @@ kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/main/buil
 
 The following configuration parameters have to be provided through the Flow Aggregator
 ConfigMap. `externalFlowCollectorAddr` is a mandatory parameter. We provide an example
-value for this parameter in the following snippet. If you have deployed the ELK
-flow collector, then please use the address, `<Logstash Cluster IP>:4739:UDP`.
+value for this parameter in the following snippet.  
+
+* If you have deployed the ELK
+flow collector, then please use the address:  
+`<Logstash Cluster IP>:4739:UDP`
+* If you have deployed the ipfix-collector, then please use the address:  
+`<Ipfix-Collector Cluster IP>:<port>:<TCP|UDP>`
```

Comment:
maybe add a markdown link to the appropriate section for each collector, using the auto-generated anchors for markdown headings